src/main/java/com/booking/superride/controller/TaxiController.java

```diff
@@ -21,12 +20,14 @@ public class TaxiController {
     private final TaxiService taxiService;
 
     @PostMapping("/add-taxi")
-    public ResponseEntity<TaxiDetailsDTO> saveTaxiDetails(@RequestBody @Valid AddTaxiRequest addTaxiRequest) {
+    public ResponseEntity<TaxiDetailsDTO> saveTaxiDetails(

```

Comment:
As a good practice avoid using public unless it is necessary. All methods and class can also be declared as package private